services/loadbalancer/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.10.2 (2024-04-11)
+
+- Update `core` to [`v0.12.0`](../../core/CHANGELOG.md#v0120-2024-04-11)
```

Comment:
Isn't it missing here two entrys?
```
- Set config.ContextHTTPRequest in Execute method
- Support WithMiddleware configuration option in the client
```

Reply:
And then it should be minor

Reply:
This was discussed with @joaopalet. The changes in the CHANGELOG were not really applied since I reverted the entire api_default.go changes due to the missing beautification on their end. 
When João merges these changes he will update the versions and changelogs as well as the code. The changes to the client.go should also be reverted then, since without setting the cfg.Middleware, they do nothing.
But what do you think?